src/volumesnapshot/templates/clusterrole.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ template "volumesnapshot.fullname" . }}
+  labels:
+    app: {{ template "volumesnapshot.name" . }}
+    chart: {{ template "volumesnapshot.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+  - apiGroups:
+      - "volumesnapshot.external-storage.k8s.io"
+    resources:
+      - "*"
+    verbs:
+      - "*"
+  - apiGroups:
+      - "apiextensions.k8s.io"
+    resources:
+      - customresourcedefinitions
+    verbs:
+      - "create"
+  # Provisioner only
+  - apiGroups:
+      - "storage.k8s.io"
+    resources:
+      - storageclasses
+    verbs:
+      - list
```

Comment:
there are couple of rules missing, see:
https://github.com/drekle/charts-1/pull/1

Reply:
merged